core/src/components/menu/menu.scss

```diff
@@ -83,12 +83,12 @@
       * Otherwise, the content will have less space on both sides.
       *
       * RTL:
-      * The right side of the menu touches the screen edge. The safe area padding has
-      * already been set in the core styles, so there's no need to set it again.
+      * The right side of the menu touches the screen edge.
+      * The right side needs to revert back to the default value that was set in the core styles, so we unset it here.
```

Comment:
Can you add details to this comment explaining why we need to do this? Ditto for the other comment below.